src/account/serialization.js

```diff
@@ -151,7 +173,15 @@ export const fromOperationRaw = (
     extra,
     subOperations,
     internalOperations,
-    ...op
+    id,
+    hash,
+    type,
+    senders,
+    recipients,
+    blockHeight,
+    blockHash,
+    transactionSequenceNumber,
+    hasFailed,
```

Comment:
`hasFailed` and `transactionSequence` was optional, is that ok that we got `hasFailed: undefined` or `transactionSequenceNumber: undefined` on all operation object (see snapshot result) ?

Reply:
I think it's ok, unless we use `"transactionSequenceNumber" in t` anywhere

Reply:
i'll see if easy to optim